api/src/main/java/org/openmrs/module/fgh/mpi/SnapshotEventProcessor.java

```diff
@@ -128,27 +142,83 @@ public void process(DatabaseEvent event) {
 				});
 				
 				if (!fhirPatients.isEmpty()) {
-					Map<String, Object> fhirBundle = new HashMap(3);
-					fhirBundle.put(MpiConstants.FIELD_RESOURCE_TYPE, MpiConstants.BUNDLE);
-					fhirBundle.put(MpiConstants.FIELD_TYPE, MpiConstants.BATCH);
-					fhirBundle.put(MpiConstants.FIELD_ENTRY, fhirPatients);
+					int successPatientCount = 0;
 					
-					List<Object> response = mpiHttpClient.submitBundle(mapper.writeValueAsString(fhirBundle));
-					//The response is a list of 2 MPI identifiers for each patient
-					int successPatientCount = response.size() / 2;
-					if (fhirPatients.size() == successPatientCount) {
-						if (log.isDebugEnabled()) {
-							log.debug("All patients in the batch were successfully processed by the MPI");
+					if (mpiContext.getMpiSystem().isOpenCr()) {
```

Comment:
This code is not correct, it breaks the initial loading logic, e.g. there is code that common regardless of the MPI system that you have moved and won't execute for Sante, the only thing that is specific to the MPIU system is how to parse and inteeprete the response otherwise the rest of the code applies to both.